providers/src/airflow/providers/edge/worker_api/datamodels.py

```diff
@@ -67,3 +82,10 @@ class WorkerQueueUpdateBody(BaseModel):
     remove_queues: Optional[List[str]] = Field(  # noqa: UP006, UP007 - prevent pytest failing in back-compat
         description="Queues to remove from worker."
     )
+
+
+class PushLogsBody(BaseModel):
+    """Incremental new log content from worker."""
+
+    log_chunk_time: datetime = Field(description="Time of the log chunk at point of sending.")
+    log_chunk_data: str = Field(description="Log chunk data as incremental log text.")
```

Comment:
Same as "Part1" PR -- you can move this to `Annotated` and make metadata part of typehints instead

Reply:
Did not we already drop it  and switched to fast_api ?